src/components/ItemTile/ItemTile.module.scss

```diff
@@ -72,6 +72,9 @@
     align-items: stretch;
     height: 50vh;
   }
+  .container:last-child {
```

Comment:
Po co to w pr'ce na widok mobilny, skoro to do desktopu?

Reply:
usuniete